### PR TITLE
Fixed #28810 -- Use the Python 3-esque str.format logging formatting style over '%'.

### PR DESCRIPTION
**Pull request by lamby**

### CODE REVIEW

**Comment by lamby** [comment]
https://code.djangoproject.com/ticket/28810

File: django/utils/log.py [hunk 1]
```diff
@@ -27,7 +27,8 @@
     'formatters': {
         'django.server': {
             '()': 'django.utils.log.ServerFormatter',
-            'format': '[%(server_time)s] %(message)s',
+            'format': '[{server_time} {message}',
```
Comment (hvdklauw) [inline django/utils/log.py]: You lost a ] in here.

Comment (pope1ni) [inline django/utils/log.py]: Tests are failing because you dropped the `]`.

File: docs/topics/logging.txt [hunk 2]
```diff
@@ -299,10 +299,12 @@ Finally, here's an example of a fairly complex logging setup::
         'disable_existing_loggers': False,
         'formatters': {
             'verbose': {
-                'format': '%(levelname)s %(asctime)s %(module)s %(process)d %(thread)d %(message)s'
+                'format': '{levelname} {asctime} {module} {process} {thread} {message}'
```
Comment (pope1ni) [inline docs/topics/logging.txt]: You should use `{process:d}` and `{thread:d}` here otherwise you are changing the behaviour.

Comment (pope1ni) [reply docs/topics/logging.txt]: Also add a trailing comma.

File: docs/topics/logging.txt [hunk 3]
```diff
@@ -299,10 +299,12 @@ Finally, here's an example of a fairly complex logging setup::
...
+                'style': '{',
             },
             'simple': {
-                'format': '%(levelname)s %(message)s'
+                'format': '{levelname} {message}'
```
Comment (pope1ni) [inline docs/topics/logging.txt]: Add a trailing comma.

File: django/utils/log.py (hunk 1 above)
Comment (pope1ni) [reply django/utils/log.py]: Hah. You just got ahead of me!

File: docs/topics/logging.txt (hunk 3 above)
Comment (lamby) [reply docs/topics/logging.txt]: Updated.

File: docs/topics/logging.txt (hunk 2 above)
Comment (lamby) [reply docs/topics/logging.txt]: Updated.

File: django/utils/log.py (hunk 1 above)
Comment (lamby) [reply django/utils/log.py]: Updated.

Comment (lamby) [reply django/utils/log.py]: Updated.

**Pull request merged**